packages/discovery-provider/src/queries/get_track_comment_count.py

```diff
@@ -0,0 +1,70 @@
+from src.models.comments.comment import Comment
+from src.models.moderation.muted_user import MutedUser
+from src.models.tracks.track import Track
+from src.models.comments.comment_report import (
+    COMMENT_REPORT_KARMA_THRESHOLD,
+    CommentReport,
+)
+from sqlalchemy import and_, func, or_
+from src.models.users.aggregate_user import AggregateUser
+from src.utils.db_session import get_db_read_replica
+
+
+def get_track_comment_count(track_id, current_user_id, db_session=None):
```

Comment:
nice work on this. i wonder if we can share code at some point

Reply:
Yeah I had the same thought but I didn't want to get stuck in a big refactor at this point in the game. Ideally we share the logic between get comments, get counts, and get replies